src/util/create-hyperlink.ts

```diff
@@ -39,7 +40,7 @@ function toLegacyQuery(params: ParamsWithKindAndStory): { [key: string]: string
 function createStoryHyperlink(
     baseUrl: string,
     params: StoryHyperlinkParams,
-    options: StoryHyperlinkOptions = { format: "old" }
+    options: StoryHyperlinkOptions = { format: "old", useDocsPage: false }
```

Comment:
I'm not sure but I think default value of `useDocsPage` should be true, for backward compatibility.

Reply:
DocsPage support was added on the last release and it was a breaking change (not intended to be). This PR tries to mitigate its breaking behavior, hence the default is false.

Reply:
👌